packages/vite/src/plugins/plugin.ts

```diff
@@ -365,6 +365,7 @@ function getOutputs(
 
   const isBuildable =
     build?.lib ||
+    build?.rollupOptions?.input ||
     build?.rollupOptions?.inputs ||
```

Comment:
Is `inputs` even a valid RollupOption? https://rollupjs.org/configuration-options/#input